packages/xdl/src/modules/config.js

```diff
@@ -205,6 +205,11 @@ const expoSdkUniversalModules = [
     libName: 'expo-font',
     sdkVersions: '>=30.0.0',
   },
+  {
+    podName: 'EXLocalization',
+    libName: 'expo-localization',
+    sdkVersions: '>=31.0.0',
+  },
```

Comment:
Can we keep this array sorted alphabetically by `libName`? 😄 Sorry for nitpicking 😅